WSFrontend/org.openl.rules.ruleservice.ws/src/org/openl/rules/ruleservice/rest/AdminRestController.java

```diff
@@ -6,38 +6,43 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-import javax.annotation.Resource;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
 import org.openl.info.OpenLVersion;
 import org.openl.info.SysInfo;
 import org.openl.rules.ruleservice.loader.DeployClasspathJarsBean;
 import org.openl.rules.ruleservice.servlet.ServiceInfo;
 import org.openl.rules.ruleservice.servlet.ServiceInfoProvider;
 
+@Service

```

Comment:
Redundant annotation, because there is no component scanning for `org.openl.rules.ruleservice.rest` package.